app/services/analytics_events.rb

```diff
@@ -1442,13 +1458,24 @@ def idv_proofing_resolution_result_missing(proofing_components: nil, **extra)
 
   # User submitted IDV password confirm page
   # @param [Boolean] success
+  # @param [Boolean] fraud_review_pending
+  # @param [Boolean] fraud_rejection
   # @param [Idv::ProofingComponentsLogging] proofing_components User's current proofing components
   # @param [String, nil] deactivation_reason Reason user's profile was deactivated, if any.
-  def idv_review_complete(success:, deactivation_reason: nil, proofing_components: nil, **extra)
+  def idv_review_complete(
+    success:,
+    fraud_review_pending: nil,
+    fraud_rejection: nil,
```

Comment:
I noticed all the other analytics methods don't have default values for the new args. If you need them here, should it be `false` rather than `nil`?